src/code42cli/cmds/users.py

```diff
@@ -60,6 +61,50 @@ def list_users(state, org_uid, role_name, active, inactive, format):
         formatter.echo_formatted_dataframe(df)
 
 
+@users.command()
+@click.argument("username")
+@click.argument("role-name")
+@sdk_options()
+def add_role(state, username, role_name):
+    """Add the specified role to the specified username"""
+    _add_user_role(state.sdk, role_name, username)
+
+
+@users.command()
+@click.argument("username")
+@click.argument("role-name")
+@sdk_options()
+def remove_role(state, username, role_name):
+    """Add the specified role to the specified username"""
+    _remove_user_role(state.sdk, role_name, username)
+
+
+def _add_user_role(sdk, role_name, username):
+    user_id = _get_user_id(sdk, username)
+    try:
```

Comment:
I don't understand what this `try` catch is doing.
It looks like it catches an error just to raise it again.
Is there any technical difference of just letting the error raise naturally?  If so, we should add a comment explaining what it is because it is not obvious.

Reply:
Also, if it is needed, it could be a helper method and `_remove_user_role()` can share it

Reply:
The try/except block isn't doing anything, actually - I will remove it.

We don't actually even need the roleID from _get_role_id to proceed. However, I like the error handling Cecilia had built for an invalid username. It handles cases where an invalid username was entered and provides the following error message: 

```
Error: Role with name Fake_Role not found.
```



Otherwise, the error is: 


```
Error: Problem making request to server.
```


Py42 doesn't really have a method for determining if a role name is valid.

Reply:
Might be worth adding a comment about that, or even extracting the error handling into a separate method
(still approved but thought I'd mention that)